example/azure-pipelines.yml

```diff
@@ -0,0 +1,147 @@
+    name: situm/sys-kubernetes-templates.git
+
+jobs:
+  #===============================================================
+  # Job for Android app
+  #===============================================================
+  - job:
+    displayName: Build Android App
+    pool:
+      vmImage: $(ubuntuVersion)
+    steps:
+    - ${{ if ne(variables['Build.Reason'],'manual') }}:
+      - bash: |
+           echo "##vso[task.setvariable variable=XAPIKEY]$(echo $(XAPIKEY_EXAMPLES))"
+           echo "##vso[task.setvariable variable=XAPIEMAIL]$(echo $(XAPIEMAIL_EXAMPLES))"
+           echo "##vso[task.setvariable variable=BUILDINGID]$(echo $(BUILDING_ID_EXAMPLES))"
+           echo "##vso[task.setvariable variable=MAPVIEWERDOMAIN]$(echo $(DEFAULT_MAPVIEWURL))"
+           echo "##vso[task.setvariable variable=REMOTEIDENTIFIER]$(echo $(DEFAULT_REMOTEIDENTIFIER))"
+           echo "##vso[task.setvariable variable=BUILDNUMBER]$(echo $(Build.BuildNumber) | awk -F"." '{if ($2 < 10) print $1"0"$2; else print $1$2}')"
+
+           echo -e "\n[!] I'm a shy step so I don't show any information. You'll see my work in the Configure Credentials step"
+        displayName: Configure Automatic run Vars
+
+    - ${{ if eq(variables['Build.Reason'],'manual') }}:
+      - bash: |
+           echo "##vso[task.setvariable variable=XAPIKEY]$(echo ${{parameters.XAPIKEY}})"
+           echo "##vso[task.setvariable variable=XAPIEMAIL]$(echo ${{ parameters.XAPIEMAIL}})"
+           echo "##vso[task.setvariable variable=BUILDINGID]$(echo ${{ parameters.BUILDINGID}})"
+           echo "##vso[task.setvariable variable=MAPVIEWERDOMAIN]$(echo ${{ parameters.MAPVIEWERDOMAIN }})"
+           echo "##vso[task.setvariable variable=REMOTEIDENTIFIER]$(echo ${{ parameters.REMOTEIDENTIFIER }})"
+           echo "##vso[task.setvariable variable=BUILDNUMBER]$(echo $(Build.BuildNumber) | awk -F"." '{if ($2 < 10) print $1"0"$2; else print $1$2}')"
+
+           echo -e "\n[!] I'm a shy step so I don't show any information. You'll see my work in the Configure Credentials step"
+        displayName: Configure Manual run Vars
+
+    - template: azure/common-steps.yaml
+      parameters:
+        XAPIKEY: $(XAPIKEY)
+        XAPIEMAIL: $(XAPIEMAIL)
+        BUILDINGID: $(BUILDINGID)
+        MAPVIEWERDOMAIN: $(MAPVIEWERDOMAIN)
+        REMOTEIDENTIFIER: $(REMOTEIDENTIFIER)
+        system: $(ubuntuVersion)
+        BUILDNUMBER: $(BUILDNUMBER)
+
+    - template: azure/build-examples.yaml
+      parameters:
+        system: $(ubuntuVersion)
+        BUILDINGID: $(BUILDINGID)
+        XAPIEMAIL: $(XAPIEMAIL)
+        MAPVIEWERDOMAIN: $(MAPVIEWERDOMAIN)
+        BUILDNUMBER: $(BUILDNUMBER)
+
+  #===============================================================
+  # Job for iOS app
+  #===============================================================
+  - job:
+    displayName: Build iOS App
+    pool:
+      vmImage: $(macOSVersion)
+    steps:
+    - ${{ if ne(variables['Build.Reason'],'manual') }}:
```

Comment:
Entendo que o concepto automatic non existe, porque ter que fixarse no ne despista un pouco

Reply:
Tal cual non porque podena facer saltar multitude de cousas entón é máis sinxelo negaro o Manual que buscar a razón porqué saltou.

Concordo en todo caso que ter que buscar o "ne" complica